bindings/python/dlite-storage.i

```diff
@@ -49,8 +49,12 @@ Returns next instance or None if exhausted.") next;
   %newobject next;
   struct _DLiteInstance *next(void) {
     char uuid[DLITE_UUID_LENGTH+1];
-    if (dlite_storage_iter_next($self->s, $self->state, uuid) == 0)
-      return dlite_instance_load($self->s, uuid);
+    if (dlite_storage_iter_next($self->s, $self->state, uuid) == 0) {
+      DLiteInstance *inst = dlite_instance_load($self->s, uuid);
+      // Why isn't the refcount already increased?
```

Comment:
Is this a potential bug somewhere?

Reply:
Good point. Created a new issue about it: #647